app/views/games/categories/show.slim

```diff
@@ -1,20 +1,54 @@
 - title @game.name
 - content_for(:header) do
-  ol.breadcrumb
+  ol.breadcrumb.shadow
+    li.breadcrumb-item = link_to "Splits I/O", root_path
     li.breadcrumb-item = link_to "Games", games_path
     li.breadcrumb-item = link_to @category.game, game_path(@category.game)
     - unless @on_game_page
       li.breadcrumb-item = link_to @category, game_category_path(@game, @category)
-  h3 = @category.game
+  h1 = @category.game
+  - if @game.aliases.where.not(name: @game.name).present?
+    h6 Also known as:
+    ul
+      - @game.aliases.where.not(name: @game.name).each do |game_alias|
+        li = game_alias
   h6
     ul.run-options
+      - if @game.srl_uri.present?
+        li = link_to image_tag(asset_path("srl.png")), @game.srl_uri
       - if can?(:edit, @game)
-        li = link_to 'edit', edit_game_path(@game)
-  - if @game.srl_uri.present?
-    = link_to image_tag(asset_path("srl.png")), @game.srl_uri
+        li
+          a.btn.btn-outline-success href=edit_game_path(@game)
+            i.fas.fa-edit>
+            span Edit
 
 article data-turbolinks-temporary=true
-  = render partial: 'shared/category_tabs', locals: {game: @game, current_category: @category, link_type: :normal}
-  = render partial: 'shared/run_table', locals: table_locals(:category, category: @category)
+  .row
+    .col-sm-3.mb-3
+      .statcard.p-3
+        h3.statcard-number = @game.users.count
```

Comment:
This isn't a concern right now, but if this page becomes a performance problem, it might be worth looking into cache counters for the model.  Essentially this tells rails to track how many of a relation exist all the time and store it in the db.